src/elements/emby-scroller/emby-scroller.scss

```diff
@@ -3,9 +3,7 @@
 }
 
 .emby-scroller {
-    padding-left: 3.3%;
     padding-left: max(env(safe-area-inset-left), 3.3%);
-    padding-right: 3.3%;
     padding-right: max(env(safe-area-inset-right), 3.3%);
```

Comment:
These are for backward compatibility. Revert please.

Reply:
Sorry thought your original comment was suggesting to remove that. Reverted.